src/Widgets/DataCountWidget/DataCountWidgetComponent.tsx

```diff
@@ -36,3 +70,35 @@ const Count = connect(
     ),
   },
 )
+
+function tag(
+  level: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | null,
+  style:
+    | 'display-1'
+    | 'display-2'
+    | 'display-3'
+    | 'display-4'
+    | 'display-5'
+    | 'display-6'
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
+    | 'text-small'
+    | 'body-font-size',
+): keyof JSX.IntrinsicElements {
+  if (level) return level
+
+  if (style === 'display-1') return 'h1'
+  if (style === 'display-2') return 'h2'
+  if (style === 'display-3') return 'h3'
+  if (style === 'display-4') return 'h4'
+  if (style === 'display-5') return 'h5'
+  if (style === 'display-6') return 'h6'
+  if (style === 'text-small') return 'div'
+  if (style === 'body-font-size') return 'div'
```

Comment:
* [ ] FYI We now have three slightly diverging implementations of `tag()`.
  * [ ] Could we use the "style" values of the HeadlineWidget for simplicity?
  * [ ] FYI unrelated: why is BannerHeadline not on par with Headline?

Reply:
Thinking a bit more about it: I don't see the need to be able to specify the tag for a DataCountWidget. A `<div>` should be good enough. I'll remote the `tag` option.

Reply:
> FYI unrelated: why is BannerHeadline not on par with Headline?

I think BannerHeadline was overlooked when changes to Headline where done.